integration/integration_test.go

```diff
@@ -89,16 +91,21 @@ func TestMain(m *testing.M) {
 
 	initCommands := []*command{
 		deployInspektorGadget,
-		deploySPO,
 		waitUntilInspektorGadgetPodsDeployed,
 		waitUntilInspektorGadgetPodsInitialized,
 	}
 
+	if *deploySPOFlag {
+		initCommands = append(initCommands, deploySPO)
+	}
+
```

Comment:
What about decoupling `deploySPOFlag` from `doNotDeploy`? With this PR, the user cannot only deploy the SPO without deploying IG. I would suggest making them independent.

Reply:
Makes sense, it was a bit complicated but I think it's fine now.

Reply:
Yeah, it is clearer now. Thanks for managing this!